components/ProgressBars/ProgressBar.styled.js

```diff
@@ -2,11 +2,15 @@ import styled from "styled-components";
 
 export const ProgressbarContainer = styled.div`
   grid-area: 1/ 2 /3 / 4;
+
+  justify-self: center;
   height: 300px;
   width: 300px;
+
   @media screen and (min-width: 768px) {
     height: 400px;
     width: 400px;
+    /* padding-top: 10%; */
```

Comment:
Do we need this in `main` if the code has been commented out? I think it might be better to remove it.